packages/app-admin-ui/client/pages/Home/index.js

```diff
@@ -82,8 +82,19 @@ class HomePage extends Component {
 const ListProvider = ({ getListByKey, listKeys, ...props }) => {
   // TODO: A permission query to limit which lists are visible
   const lists = listKeys.map(key => getListByKey(key));
-  const query = gqlCountQueries(lists);
 
+  if (lists.length === 0) {
+    return (
+      <Fragment>
+        <DocTitle>Home</DocTitle>
+        <PageError>
+          <p>No lists defined. Get started by creating your first list.</p>
```

Comment:
Can this link to the docs?

```suggestion
          <p>No lists defined. <Link href="SOMEWHERE">Get started by creating your first list.</Link></p>
```

Reply:
Sure. Added.